### PR TITLE
VoIP doc: move from CRM to General + new doc for Axivox as VoIP provider

### PR DESCRIPTION
**Pull request by jcs-odoo**

### CODE REVIEW

File: general.rst [hunk 1]
```diff
@@ -11,3 +11,4 @@ General
    general/unsplash
    general/base_import
    general/auth
+   general/voip
```
Comment (jcs-odoo) [inline general.rst:14]: For the forward port in ulterior version, this line can be put after **Payment Acquirers** and before **In-App Purchase**

**Review by AntoineVDV — APPROVED** [review]
> [MOV] general: moving VoIP docs from "CRM" to "general" 

Please only do that in >= 13.0
The structure in 12.0 is too much different from that of 13.0. That'd lead to overly complicated gitmnastic and content loss.

**Review by AntoineVDV — CHANGES_REQUESTED** [review]
Clicked the wrong radio ><

**Comment by jcs-odoo** [comment]
Duplicate of #784 

which is based on 13.0

**Comment by jcs-odoo** [comment]
@AntoineVDV I guess we can delete the branch as well, then.

**Pull request closed without merging**